src/framework/entity.js

```diff
@@ -27,6 +27,9 @@ import { Application } from './application.js';
 /** @typedef {import('./components/sound/component.js').SoundComponent} SoundComponent */
 /** @typedef {import('./components/sprite/component.js').SpriteComponent} SpriteComponent */
 
+/** @type {GraphNode[]} */
```

Comment:
You'll need an @ignore here or this symbol will actually appear in the API reference (in the `pc.` namespace).

Reply:
done. I assumed those are exposed in docs only if there's a description, not only a type.